doc/rst/usingchapel/compiling.rst

```diff
@@ -58,16 +58,17 @@ Flags                   Description
 ``-O``                  turns on optimization of the generated C code
 ``--fast``              turns on ``--no-checks``, ``-O``, and enables
                         many other optimizations
+``-s <name[=expr]>``    set a config variable with the given expression
```

Comment:
It's been awhile since I've looked at this—while you're here, what do you think about changing "variable" -> "declaration" (since it applies to vars, consts, params, types)